data_enrichment.py

```diff
@@ -45,6 +48,14 @@ def format_nom_complet(
         name = f"{prenom if prenom else ''}{formatted_name}"
     if nom_raison_sociale:
         name = nom_raison_sociale
+    if denomination_usuelle_1 or denomination_usuelle_2 or denomination_usuelle_3:
```

Comment:
not sure about this piece of code. If one of `denomination_usuelle` is `None` it won't work.

Reply:
It's an `or` condition  so it works if at least one is not `None`.

Reply:
Yes. I noticed it, but I thought it was highly unlikely to have `denomination_3` and not `denomination_2`, so I chose to ignore it.
But you're right, I should also prepare for this possibility.